rts/Game/UnsyncedGameCommands.cpp

```diff
@@ -1375,15 +1375,31 @@ class TrackActionExecutor : public IUnsyncedActionExecutor {
 	TrackActionExecutor() : IUnsyncedActionExecutor("Track",
 			"Start/stop following the selected unit(s) with the camera", false, {
 			{"", "Toggles tracking"},
-			{"<on|off>", "Set tracking <on|off>"},
+			{"<on|off>", "Set tracking <on|off> <unitID1> <unitID..N>"},
```

Comment:
It looks like the first unitID is somehow special because it's in a separate \<bracket\> but I don't think it has any different treatment? Consider a single \<bracket\>
```suggestion
			{"<on|off>", "Set tracking <on|off> <unitID, unitID...>"},
```